js/ui/scroll_view/ui.scrollbar.js

```diff
@@ -216,13 +227,19 @@ var Scrollbar = Widget.inherit({
                 this.option()[args.name] = this._normalizeSize(args.value);
                 this._update();
                 break;
+            case "baseContentSize":
```

Comment:
This options have no default value. If it is required to add them, they should be defined in _defaultOptions method and tested in defaultOptions.tests.js

It is strongly recommended don't add any new options or flags to the code for bug fixing. The new option can be added when there is no way to fix a bug without it.

Reply:
> This options have no default value. If it is required to add them, they should be defined in _defaultOptions method and tested in defaultOptions.tests.js
> 
> It is strongly recommended don't add any new options or flags to the code for bug fixing. The new option can be added when there is no way to fix a bug without it.

It's possible to solve the task without adding new options, but _getBaseDimension function will be dupclicated in two files. Is it better than adding a new option?

Reply:
No, duplication is bad idea too. Please try to find better solution

Reply:
I agree with @mrjohn0011. Why these options are used in the processing of the options changed if none updates via the option method outside? And I don't see **tests** on these changes!